components/x11/ftgl/ftgl.p5m

```diff
@@ -0,0 +1,57 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2018 Harry Liebel <hliebel@gmail.com>
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=pkg.description value="$(COMPONENT_DESCRIPTION)"
+set name=pkg.human-version value=$(HUMAN_VERSION)
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+# gmake REQUIRED_PACKAGES cant resolve libGL
+<transform file -> add pkg.depend.bypass-generate libGL\.so\.1>
+
+file path=usr/include/FTGL/FTBBox.h
+file path=usr/include/FTGL/FTBitmapGlyph.h
+file path=usr/include/FTGL/FTBuffer.h
+file path=usr/include/FTGL/FTBufferFont.h
+file path=usr/include/FTGL/FTBufferGlyph.h
+file path=usr/include/FTGL/FTExtrdGlyph.h
+file path=usr/include/FTGL/FTFont.h
+file path=usr/include/FTGL/FTGLBitmapFont.h
+file path=usr/include/FTGL/FTGLExtrdFont.h
+file path=usr/include/FTGL/FTGLOutlineFont.h
+file path=usr/include/FTGL/FTGLPixmapFont.h
+file path=usr/include/FTGL/FTGLPolygonFont.h
+file path=usr/include/FTGL/FTGLTextureFont.h
+file path=usr/include/FTGL/FTGlyph.h
+file path=usr/include/FTGL/FTLayout.h
+file path=usr/include/FTGL/FTOutlineGlyph.h
+file path=usr/include/FTGL/FTPixmapGlyph.h
+file path=usr/include/FTGL/FTPoint.h
+file path=usr/include/FTGL/FTPolyGlyph.h
+file path=usr/include/FTGL/FTSimpleLayout.h
+file path=usr/include/FTGL/FTTextureGlyph.h
+file path=usr/include/FTGL/ftgl.h
+file path=usr/lib/$(MACH64)/libftgl.a
```

Comment:
Why do we deliver static library?

Reply:
Its just a install product of the default library build. What if someone wants to statically link against this library when building something else? I see in /var/lib that a few other libraries do the same (like boost, sdl2, qt)..

Reply:
We shouldn't encourage user to do silly things (like static linking). Think about library updates. We deliver static variants of libraries when there's no dynamic version, but generally static libs shouldn't be shipped until strictly necessary.